### ISSUE TITLE
Upgrade from 17.2 to 18.0 failed.

### ISSUE THREAD

**Issue by melihinanpak** [solvsoftware/Ghost-Azure#13]
Hi there,

Thanks for your effort and support to Azure platform.
My issue is related to upgrading (17.2 to 18.0).

I have a Ghost 17.2 on the Azure Web App and working well.
But, I initiating staging deployment slot with your repository will fail.

How can I do for my web app upgrade from your repository?

**Comment by RadoslavGatev** [comment]
Hi,
I will test this over the weekend.

**Comment by melihinanpak** [comment]
any update?

**Comment by RadoslavGatev** [comment]
Hey,

It took me a while until I was able to pick it up.
I found no issues when upgraded 17.2 to 18.0 on a test web app.

I took the following steps:
1. Deployed 17.2 to a new web app
2. Executed `node db.js` in order to create a database
3. Deployed 18.0 
4. Make sure `WEBSITE_NODE_DEFAULT_VERSION` app setting is set to `8.9.0`
5. Executed `npm rebuild` to rebuild the packages to be used with the newer version of Node.js
6. Executed the modified version of db.js in order to update the database
```javascript
var KnexMigrator = require('knex-migrator');
var knexMigrator = new KnexMigrator({
    knexMigratorFilePath: __dirname
});

//knexMigrator.init();
knexMigrator.migrate();
```
Happy New Year!!
